userguide/vms.rst

```diff
@@ -405,6 +405,9 @@ When a VM is already running, these buttons are available:
   VM must have a VNC device and :guilabel:`VNC Web` enabled in that
   device.
 
+Each VM generates a specific log file. The log files for VMs are in
```

Comment:
Same as above.  I'd guess each VM has a subdirectory in /var/logs/vm/, but this does not make that clear.